### ISSUE TITLE
Android usage example

### ISSUE THREAD

**Issue by xabi-nix** [sqvcnet/libvr#1]
Hello! Are there any Android usage examples?
Thanks.

**Comment by sqvcnet** [comment]
I need some time to commit android sample, maybe two days later.
Now I am too busy to do that.


原始邮件
发件人:xabi-nixnotifications@github.com
收件人:sqvcnet/libvrlibvr@noreply.github.com
抄送:Subscribedsubscribed@noreply.github.com
发送时间:2017年12月27日(周三) 18:26
主题:[sqvcnet/libvr] Android usage example (#1)


Hello! Are there any Android usage examples?
 Thanks.

**Comment by xabi-nix** [comment]
It would be great! Thanks in advance!

**Comment by sqvcnet** [comment]
Have you try thishttps://github.com/sqvcnet/react-native-libvr-democross platfrom solution by yourself ?
This project can run on iOS and Android used react-native as a cross platfrom framework.


原始邮件
发件人:xabi-nixnotifications@github.com
收件人:sqvcnet/libvrlibvr@noreply.github.com
抄送:sqvcnetshanqiang888@sina.com; Commentcomment@noreply.github.com
发送时间:2017年12月28日(周四) 18:30
主题:Re: [sqvcnet/libvr] Android usage example (#1)


It would be great! Thanks in advance!

**Comment by xabi-nix** [comment]
Yes, but it can be used only for video playback. FFmpeg version is more scalable.
